### PR TITLE
Added admin to basic controller contract

### PR DESCRIPTION
**Pull request by MattCharles**

### CODE REVIEW

File: contracts/BasicController.sol [hunk 1]
```diff
@@ -17,21 +17,24 @@ contract BasicController is
...
   }
 
-  function initialize(IRegistrar _registrar) public initializer {
+  function initialize(IRegistrar _registrar, address _admin) public initializer {
     __ERC165_init();
     __Context_init();
     __ERC721Holder_init();
 
     registrar = _registrar;
+    admin = _admin;
```
Comment (Remscar) [inline contracts/BasicController.sol:45]: Nit: Might be nice to have a function where the admin can transfer admin-ship to another address

Comment (MattCharles) [reply contracts/BasicController.sol:45]: Added this

**Review by JamesEarle — APPROVED** [review]
AkshayKumarVeryNiceGiftOfFbOkayGIF

**Review by RyanCross — APPROVED** [review]
👍

**Pull request merged**